print/components/story/StoryDay.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="tw-mb-4">
+    <div class="tw-text-2xl tw-mb-2 tw-break-after-avoid">
+      <h1>{{ $tc('entity.day.name') }} {{ day.number }} ({{ dateLong(day.start) }})</h1>
+    </div>
+
+    <template v-if="entriesWithStory.length">
+      <template v-for="{ scheduleEntry, storyChapters } in entriesWithStory">
+        <div v-for="chapter in storyChapters" :key="chapter._meta.uri" class="tw-mb-3">
+          <h4 class="tw-text-lg tw-font-bold tw-break-after-avoid">
+            <span class="d-inline-flex align-center">
+              <span>{{ scheduleEntry.number }}</span>
+              <category-label
+                :category="scheduleEntry.activity().category()"
+                class="tw-ml-1"
+              />
+            </span>
+
+            <span>{{ scheduleEntry.activity().title }}</span>
+            <template v-if="chapter.instanceName">
+              - {{ chapter.instanceName }}
+            </template>
+          </h4>
+
+          <rich-text :rich-text="chapter.text" />
+        </div>
```

Comment:
Can you please add `break-inside: avoid` so that heading and content are not on different pages?

Reply:
There's already a `tw-break-after-avoid` on the `h4` tag. I think that's closer to what we want than `break-inside-avoid`. 

Only working in pagedjs though and not in chromium native. Should come with LayoutNGPrinting, though (https://bugs.chromium.org/p/chromium/issues/detail?id=223068)